geomstats/backend/__init__.py

```diff
@@ -10,6 +10,8 @@
 
 _BACKEND = _backend
```

Comment:
Out of curiosity: why the _ before BACKEND?

Reply:
private